schemas/Bills/tradeBill.yaml

```diff
@@ -23,72 +23,159 @@ paths:
 
 components:
   schemas:
+    ###########
+    #
+    # Events
+    #
+    ###########
+    
     BillEvent:
       type: object
       required:
         - bill
-        - project
-        - lineItems
-        - approval
         - eventMetadata
       properties:
         bill:
           $ref: '#/components/schemas/Bill'
-        project:
-          $ref: '#/components/schemas/Project'
-        lineItems:
-          type: array
-          items:
-            $ref: '#/components/schemas/LineItem'
-        approval:
-          $ref: '#/components/schemas/Approval'
         eventMetadata:
           $ref: '#/components/schemas/Metadata'
-
+          
+    ###########
+    #
+    # Enums
+    #
+    ###########
+    
+    BillType:
+      type: string
+      enum: 
+        - STANDARD
+        - DEPOSIT
+        - OVERHEAD
+        - REPAIR_AND_MAINTENANCE
+        - WARRANTY    
+        
+    BillSource:
+      type: string
+      enum: 
+        - MANUAL
+        - CLICK_TO_PAY
+        - AUTO_PARSED
+        
+    BillStatus:
+      type: string
+      enum:
+        - DRAFT
+        - CHANGES_REQUESTED
+        - RELEASE_SCHEDULED
+        - PENDING_APPROVAL
+        - READY_TO_PAY
+        - REJECTED
+        - PAID
+        - REVERSED
+        - CANCELLED
+        - PENDING_CONDITIONAL_LIEN_WAIVER
+        - PENDING_UNCONDITIONAL_LIEN_WAIVER
+        - UNRELEASED
+        - PARTIALLY_PAID
+        
+    LotType:
+      type: string
+      enum:
+        - BOYL
+        - BOOL
+        - HBL
+        - BTR
+        - FEE
+    
+    ProjectStatus:
+      type: string
+      enum:
+        - ACTIVE
+        - COMPLETED
+        - CLOSED
+        - HOLD
+        - UNDER_WARRANTY
+    
+    ApprovalStatus:
+      type: string
+      enum:
+        - CHANGES_NEEDED
+        - REQUESTED
+        - APPROVED
+        - REJECTED
+    
+    CostClassificationType:
+      type: string
+      enum:
+        - SOFT
+        - HARD
+      
+    ###########
+    #
+    # Objects
+    #
+    ###########
+    
     Bill:
       type: object
       required:
-        - billId
+        - id
+        - type
+        - source
+        - status
+        - project
+        - lineItems
+        - approval
+        - tradePartner
         - billNumber
         - tradePartnerBillNumber
-        - tradePartnerId
-        - billType
-        - billSource
         - billDate
-        - billStatus
-        - totalAmountInCents
+        - amountInCents
       properties:
-        billId:
-          type: string
-        billNumber:
-          type: string
-        tradePartnerBillNumber:
-          type: string
-        tradePartnerId:
-          type: string
-        tradePartnerName:
-          type: string
-        billType:
+        id:
```

Comment:
each entity in our backend just has its own `id` field, it isn't prefixed with the entity name.  We also have a standard format for all IDs, so i thought it might be useful to use that here.